fission-server/src/extract/authority.rs

```diff
@@ -0,0 +1,143 @@
+//! Authority extractor
+
+use axum::{
+    async_trait,
+    extract::{FromRequestParts, TypedHeader},
+    headers::{authorization::Bearer, Authorization},
+    http::request::Parts,
+    RequestPartsExt,
+};
+
+use ucan::ucan::Ucan;
+
+// 🧬
+
+use crate::authority::Authority;
```

Comment:
@zeeshanlakhani I've noticed that in some places `fission_server::*` works, but not here, why is that?

Reply:
The correct form is using crate though for internal mods.